lib/transmission-rss/client.rb

```diff
@@ -12,6 +12,9 @@ class Client
     class Unauthorized < StandardError
     end
 
+    class E429 < StandardError
```

Comment:
rename `E429` to `TooManyRequests`, please

Reply:
Done